src/open_inwoner/accounts/models.py

```diff
@@ -313,7 +313,8 @@ def seed(self):
         return self._seed
 
     def get_full_name(self):
-        parts = (self.first_name, self.infix, self.last_name)
+        first_name = self.display_name or self.first_name
```

Comment:
The `display_name` field uses `CharFieldValidator`, which allows strings containing only spaces. We probably don't want to prefer a display_name containing only space. We could should `strip()` and use `isspace()` to check for that or change the validator on the model field to prevent this from arising in the first place.

Reply:
Ugh that's not great. It is unlikely we have only-spaces values because form-fields use .strip() by default but I modified the code to be safe.